controllers/businesses-controller.js

```diff
@@ -0,0 +1,166 @@
+const mongoose = require('mongoose')
+
+const business = require('../models/business')
+
+// Create Business
+const createBusiness = async (req, res, next) => {
+  const {
+    _id,
```

Comment:
see what Aderson has to say about taking out or leaving _id, userId, createDate (all of these autoGenerated?) and should status have a default? (line 129). maybe for now - both this and user registration have variables i.e. active, not active for here, user or owner or registration...

Reply:
Thank you for your insights @cynthiaeddy!  Per our slack convo, from the createBusiness function I will remove the auto-generated businesses collection's `_id` and `userId` that references the auto-generated `_id` from the users collection. However, since `createDate` is not auto generated it will remain.